### ISSUE TITLE
req.body parse broken

### ISSUE THREAD

**Issue by corbanb** [balderdashy/skipper#68]
I have a form with multiple inputs for a user to upload an image and create their profile. When I use `multipart/form-data` the form only gets the data before the file input. After the file input req.body's data is empty. I see the right info in chrome's console as well. 

Note: this is happening irregularly as well.

sails - 10.5
skipper - 0.5.5

**Comment by Nox-404** [comment]
I got the same, please fix this :+1:

**Comment by metzuda** [comment]
@corbanb @Nox-404 

This is not an issue. From the docs : 

> user agents send any text parameters before the first file parameter in the multipart HTTP request body

Source: https://github.com/balderdashy/skipper#background

**Comment by corbanb** [comment]
@metzuda that is all good and well but doesn't always seem to work. I dug deeper and found this - #71

**Comment by sailsbot** [comment]
Thanks for posting, @corbanb.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
